src/main/java/org/prebid/server/handler/NotificationEventHandler.java

```diff
@@ -26,107 +34,172 @@
  */
 public class NotificationEventHandler implements Handler<RoutingContext> {
 
+    private static final Logger logger = LoggerFactory.getLogger(NotificationEventHandler.class);
+
+    private static final long DEFAULT_TIMEOUT = 1000L;
     private static final String TRACKING_PIXEL_PNG = "static/tracking-pixel.png";
-    private static final String TRACKING_PIXEL_JPG = "static/tracking-pixel.jpg";
-    private static final String VIEW_TYPE = "view";
-    private static final String WIN_TYPE = "win";
-    private static final String FORMAT_PARAMETER = "format";
-    private static final String TYPE_PARAMETER = "type";
-    private static final String BID_ID_PARAMETER = "bidid";
-    private static final String BIDDER_PARAMETER = "bidder";
-    private static final String JPG_FORMAT = "jpg";
-    private static final String PNG_FORMAT = "png";
-    private static final String JPG_CONTENT_TYPE = "image/jpeg";
     private static final String PNG_CONTENT_TYPE = "image/png";
 
-    private final AnalyticsReporter analyticsReporter;
-    private final Map<String, TrackingPixel> trackingPixels;
+    private static final String TYPE_PARAMETER = "t";
+    private static final String WIN_TYPE = "w";
+    private static final String IMP_TYPE = "i";
+
+    private static final String BID_ID_PARAMETER = "b";
+    private static final String ACCOUNT_PARAMETER = "a";
 
-    private NotificationEventHandler(AnalyticsReporter analyticsReporter, Map<String, TrackingPixel> trackingPixels) {
-        this.analyticsReporter = analyticsReporter;
-        this.trackingPixels = trackingPixels;
+    private static final String FORMAT_PARAMETER = "f";
+    private static final String PIXEL_FORMAT = "i";
+    private static final String BLANK_FORMAT = "b";
+
+    private static final String ANALYTICS_PARAMETER = "x";
+    private static final String ENABLED_ANALYTICS = "1";
+    private static final String DISABLED_ANALYTICS = "0";
+
+    private final AnalyticsReporter analyticsReporter;
+    private final TrackingPixel trackingPixel;
```

Comment:
Minor. Pls consider declaration order the same as in constructor initialization. It helps to read the code more easy.